### PR TITLE
Jupyterlab 2.3.0 rc0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

- for #255

### CODE REVIEW

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge/label/jupyterlab_server_rc,conda-forge,defaults
```
Comment (bollwyvl) [inline .ci_support/linux_64_.yaml:4]: this is the key change to make this a new rc branch

Comment (goanpeca) [reply .ci_support/linux_64_.yaml:4]: Cool!

**Comment by blink1073** [comment]
Thanks again @bollwyvl!

**Comment by bollwyvl** [comment]
Yeah well still haven't tested with it yet... But will this weekend.

**Pull request merged**